[artifact-manager,maven,package-installer,vro-types] (#306) Remove support for vRA and vRO 7 (#453)
vmware/build-tools-for-vmware-aria: docs/versions/latest/Release.md

[//]: # (### *Deprecation*)
[//]: # (Explain what is deprecated and suggest alternatives)

### *Deprecation of vRA 7 archetype*

> The vRA 7 Archetype and all related plugins/mojos/code is removed. As vRA 7 is Out Of Support, we won't manage it anymore.

Reviewer: ```suggestion
The vRA 7 Archetype and all related plugins/mojos/code are removed due to the fact that vRA 7 is Out Of Support.
```

> The suggested alternative is to use version 2.43.0 of the toolchain. That is the last version that will support vRA7

Reviewer: ```suggestion
The suggested alternative is to use version 2.44.0 of the toolchain. That is the last version that supports vRA7
```
